.gitignore

```diff
@@ -16,3 +16,5 @@ tmp/
 # Mac OSX Finder files.
 **/.DS_Store
 .DS_Store
+
+yarn.lock
```

Comment:
The yarn strategy for the project is not specified yet. Not sure if this is wanted.